### PR TITLE
M3-4505: CMR Table Styles for OBJ Bucket Detail

### PR DESCRIPTION
**Pull request by DevDW**
## Description
CMR table styles for table on Bucket Detail page

## Type of Change
- Non breaking change ('update', 'change')

## To-Do

- [x] Fix table styling for consistency with other tables
- [x] ObjectActionMenu

### CODE REVIEW

**Review by Jskobos — CHANGES_REQUESTED** [review]
Styles look good. Two things left:

1. Build is failing because of your earlier PR, gotta remove hideLabelOnMobile or whatever here too
2. If you have a "folder", clicking on it should "open" the "folder". It's all fake, but there doesn't seem to be any click handling at all going on in the rows. Compare test account 4 on develop and on this branch.

**Comment by Jskobos** [comment]
Maybe we'd also want to hide the "last modified" column on smDown.

**Review by WilkinsKa1 — APPROVED** [review]

File: packages/manager/src/features/ObjectStorage/BucketDetail/FolderTableRow.tsx [hunk 1]
```diff
@@ -56,17 +65,29 @@ const FolderTableRow: React.FC<Props> = props => {
...
+                </Typography>
+              </div>
+            )}
           </Grid>
         </Grid>
       </TableCell>
-      <TableCell parentColumn="Size" />
-      <TableCell parentColumn="Last Modified" />
-      {/* Empty TableCell for Action Menu (for ObjectTableRow).*/}
+      {/* Three empty TableCells corresponding to the Size, Last Modified, and Action Menu (for ObjectTableRow) columns for formatting purposes. */}
```
Comment (DevDW) [inline packages/manager/src/features/ObjectStorage/BucketDetail/FolderTableRow.tsx:82]: This also fixes a small bug in production in the folder row in mobile view–"Size" and "Last Modified" should not display in-row as table cell values:

Screen Shot 2020-09-14 at 12 53 43 PM

**Review by tiffwong — APPROVED** [review]

File: packages/manager/src/features/ObjectStorage/BucketDetail/FolderTableRow.tsx (hunk 1 above)
Comment (Jskobos) [reply packages/manager/src/features/ObjectStorage/BucketDetail/FolderTableRow.tsx:82]: 👍  good catch

**Review by Jskobos — APPROVED** [review]

**Pull request merged**